src/assets/shared/TextInputWithLabel.jsx

```diff
@@ -0,0 +1,16 @@
+function TextInputWithLabel({ elementId, label, onChange, ref, value }) {
```

Comment:
So the folder composition is slightly off. You just want `shared/TextInputWithLabel` No need to nest them in an `assets`  folder.